universal-application-tool-0.0.1/app/services/question/ScalarType.java

```diff
@@ -9,6 +10,7 @@
  * models.Applicant} JSON column and serialized using {@link services.applicant.ApplicantData}.
  */
 public enum ScalarType {
+  LIST(List.class),
```

Comment:
Question (non-blocking): Do we ever want concrete `List` types? I am curious how this works.

Reply:
Ah I fixed this in a different PR, will update here too. We will eventually want `LIST` for multi-select options (like checkboxes), but dropdowns for now are single-select. 

`ApplicantData` is set up to read/write `ImmutableList<String>` for when we do need it

Reply:
Thanks for the explanation. I was asking if we need `List<String>` or does `List` serve the need. This is just me learning Java 😅

Reply:
That's a good question! I think List will work for now, since we may have lists of different types (unlikely, but possible). If not, we could make a type reference for List<String> and get the class from that